catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -57,6 +62,10 @@
 
   private CqlTransformHandler cqlTransformHandler;
 
+  private static String LAT_LON_REGEX = "([-]?[\\d]{1,3}(\\.[\\d]+)?)|([-]?\\.[\\d]+)";
```

Comment:
✏️ Should this be final? 

You also don't need any of the square brackets in this regex, but I suppose there's an argument that it makes it more readable. `(-?\\d{1,3}(\\.\\d+)?)|(-?\\.\\d+)`